webdriver/tests/bidi/script/call_function/result.py

```diff
@@ -117,9 +117,15 @@ async def test_primitive_values(bidi_session, top_context, await_promise, expres
                 ],
             },
         ),
-        ("new WeakMap()", {"type": "weakmap"}),
-        ("new WeakSet()", {"type": "weakset"}),
-        ("new Error('SOME_ERROR_TEXT')", {"type": "error"}),
+        ("new WeakMap()", {
```

Comment:
@thiagowfx I assume this reformatting wasn't intended to get pushed?

Reply:
Especially I want to mention that we should not push further changes once a PR has been reviewed and is waiting for getting merged.

Reply:
Done (now it's consistent)